README.md

```diff
@@ -45,12 +49,21 @@ way you might write a quick commit message. Your release notes should contain a
 clear overview of all changes that were added in this release.
 
 Tags don't push up to the remote server automatically. 
-To push your tag up, use `git push --tags`. 
+To push your tag up, use `git push --follow-tags`. 
 
 For more information on tags, check https://git-scm.com/book/en/v2/Git-Basics-Tagging.
 
 ## Getting Started
 
+### Using a Tagged Release
+
+When adding scaffold to a new project, make sure you use the latest tagged release 
+instead of just cloning master (as we did previously).
+
+To grab the lastest tagged release, navigate to the 'Releases' tab for this repo and download
```

Comment:
First one's a link, yah, but what's this thing called?
![image](https://user-images.githubusercontent.com/1036605/28126911-e2fa37ca-66f8-11e7-9cda-b774d4ac03c0.png)